src/components/VBreadcrumbs/VBreadcrumbs.vue

```diff
@@ -38,17 +38,23 @@ export default {
   },
 
   computed: {
-    breadcrumbs() {
+    routeBreadcrumbs() {
```

Comment:
You cannot have a prop and a computed with the same name (the test wouldn't even load the `.vue` file with that).

So I renamed it and created the next `if` statement. Not sure if this is what the original intention was though.

Reply:
Yeah, this is mostly the result of a copy/paste job from a different project. This particular component is not being used, but I sometimes work on the DX concepts for how I imagine it being used, then hook things up to work. Haven't gotten to the "hook things up to work" part yet.